homeassistant/components/vera/__init__.py

```diff
@@ -90,8 +90,7 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
     controller.start()
 
     hass.bus.async_listen_once(
-        EVENT_HOMEASSISTANT_STOP,
-        lambda event: hass.async_add_executor_job(controller.stop),
+        EVENT_HOMEASSISTANT_STOP, lambda event: controller.stop()
```

Comment:
@vangorra, lambdas are not seen as an async function, regardless of what they return and so are already executed in the executor. So we were calling an async function from a sync context. Now it's fixed :)